scripts/website.js

```diff
@@ -28,76 +31,81 @@ function randomIntBetween(min, max) { // min and max included
 
 let loginData = {
 
-    "users": [
-        { "username": "admin", "password": "123" },
-        { "username": "test", "password": "1234" },
-        { "username": "invaliduser", "password": "password"}
-    ]
+  "users": [
+    { "username": "admin", "password": "123" },
+    { "username": "test_user", "password": "1234" },
+    { "username": "invaliduser", "password": "password"}
+  ]
 };
 
 export default function() {
-    group("Front page", function() {
-        let res = http.get("http://test.k6.io/?ts=" + Math.round(randomIntBetween(1,2000)), { tags: { name: "http://test.k6.io/?ts=*"}});
-
-
-        let checkRes = check(res, {
-            "Homepage body size is 11026 bytes": (r) => r.status === 200 && r.body.length === 11026,
-            "Homepage welcome header present": (r) => r.status === 200 && r.body.indexOf("Welcome to the k6.io demo site!") !== -1
-        });
+  group("Front page", function() {
```

Comment:
Do you know if we need a group here? It feels like we are using it just for dividing the code. Should we move to a function?

Reply:
It's not strictly needed, but this is meant to be a real-world test that uses some common k6 features, and it makes sense that some users would separate a frontpage test from a login test in this way. So I wouldn't remove `group()`.

Reply:
But it sounds like it conflicts with the discouraged group's practice https://k6.io/docs/using-k6/tags-and-groups/#discouraged-one-group-per-request

Reply:
I'm rising this topic because with the latest discussions around `group` this was one of the cases where `group` is overused. At least this was my feeling.

Reply:
But this is not doing one request per group. There are 3 requests in the `Front page` group (2 in the `Static assets` sub-group), and 2 in `Login`, along with several checks and custom metric updates. Sure, it's a bit contrived, and a proper real-world test should be cleaner, but `group()` is often used, so it makes sense to use it in a benchmark.

The goal in thise case is not to have the cleanest script, but something that uses the most common k6 features.

Reply:
I have seen it now, sorry, I misread the diff view.